2019Labs/CustomSecurityContent/documentation/lab3_oval.adoc

```diff
@@ -46,17 +46,20 @@ in the terminal.
 === OVAL introduction
 
 OVAL stands for Open Vulnerability and Assessment Language.
-It is a XML-bassed declarative language that is part of the SCAP standard, and we will focus on it's ability to query a system's state.
+In a nutshell, it is an XML-bassed declarative language that is part of the SCAP standard.
+We will focus on its ability to query and evaluate state of a system.
 Quoting from the http://ovalproject.github.io/getting-started/faqs/[OVAL FAQ]:
 
-___
-
+__
 The language standardizes the three main steps of the assessment process: representing configuration information of systems for testing; analyzing the system for the presence of the specified machine state (vulnerability, configuration, patch state, etc.); and reporting the results of this assessment.
-___
+__
+
+The `ComplianceAsCode` project uses OVAL as the language for checks.
```

Comment:
What does "a language for check" mean? Eg. uses OVAL as a language to write automated configuration checks.